.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
     - AIRFLOW_CONTAINER_MOUNT_HOST_VOLUMES="false"
     - FORCE_ANSWER_TO_QUESTIONS="yes"
     - AIRFLOW_CI_SILENT="true"
+    - DOCKERHUB_USER="potiuk" # Temporary to make it builds using images from test dockerhub before merging
```

Comment:
leaving a comment here just to make EXTRA sure this gets removed eventually :)

Reply:
Yeah. I will do some manual pushing of the images first before merge as Dockerhub is super-slow to build PROD images.

Reply:
I will add a comment myself in the new PR : @dimberman  so that I won't forget